### PR TITLE
Fix #239: Add usage with Symfony app

### LINKED ISSUE #239: Change instructions for usage with Symfony
We should add commands to Symfony app and don't use utility from bin.

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/en/usage-with-symfony.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+composer require yiisoft/db-migration
+composer require yiisoft/db-sqlite
+```
+
+Configure migrations and database connection in your `config/services.yml`:
+
+```yaml
+Yiisoft\Db\Migration\:
```
Comment (arogachev) [inline docs/en/usage-with-symfony.md:13]: Remove trailing slashes?

**Review by arogachev — CHANGES_REQUESTED** [review]

File: docs/en/usage-with-symfony.md (hunk 1 above)
Comment (xepozz) [reply docs/en/usage-with-symfony.md:13]: They're needed

File: docs/en/usage-with-symfony.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    resource: '../vendor/yiisoft/db-migration/src/'
+
+Yiisoft\Aliases\:
+    resource: '../vendor/yiisoft/aliases/src/'
```
Comment (xepozz) [inline docs/en/usage-with-symfony.md:17]: What does it mean? 
Seems like composer stuff

File: docs/en/usage-with-symfony.md (hunk 1 above)
Comment (arogachev) [reply docs/en/usage-with-symfony.md:13]: Sorry, overlooked that these are composite namespaces.

File: docs/en/usage-with-symfony.md (hunk 2 above)
Comment (samdark) [reply docs/en/usage-with-symfony.md:17]: That allows using services from the path with autowiring in Symfony DI container.

**Pull request merged**